src/chat/chatter.py

```diff
@@ -273,47 +273,76 @@ def updatePlayer(self):
         # Rating icon choice  (chr(0xB1) = +-)
         formatting = ("Global Rating: {} ({} Games) [{}\xb1{}]\n"
                       "Ladder Rating: {} [{}\xb1{}]")
-        self.rankItem.setToolTip(formatting.format(
+        tooltip_str = formatting.format(
             str(int(rating)),
             str(player.number_of_games),
             str(int(player.rating_mean)),
             str(int(player.rating_deviation)),
             str(int(ladder_rating)),
             str(int(player.ladder_rating_mean)),
-            str(int(player.ladder_rating_deviation))))
+            str(int(player.ladder_rating_deviation)))
 
         league = player.league
         if league is not None:
-            formatting = ("Division : {}\n"
-                          "Global Rating: {}")
-            self.rankItem.setToolTip(formatting.format(
-                league["division"],
-                str(int(rating))))
-            self.rankItem.setIcon(util.THEME.icon("chat/rank/%s.png" % league["league"]))
+            icon_str, tooltip_str = league["league"], "Division : " + league["division"] + "\n" + tooltip_str
```

Comment:
Was removing rating from tooltip intentional here? also format() plox :)

Reply:
guess what's in  + tooltip_str (at the end)
... and I remembered: 'format() does conversion to string for us'